services/gemma-local/benchmark.py

```
@@ -0,0 +1,269 @@
+                'tokens_generated': tokens_generated,
+                'tokens_per_second': round(tokens_per_second, 2),
+                'audio_processing_ratio': round(audio_processing_ratio, 3),
+                'response_length': len(response),
+                'response_preview': response[:100] + '...' if len(response) > 100 else response,
+                'success': True
+            }
+            
+            logger.info(f"Benchmark result: {tokens_per_second:.2f} tokens/sec, "
+                       f"{audio_processing_ratio:.3f}x realtime")
+            
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)